### PR TITLE
test(cognito-identity-test): Adding unit test coverage to StorageHelp…

### PR DESCRIPTION
**Pull request by evcodes**
…er-rn.js

#### Description of changes
Add unit test coverage to Utility class react native storage helper.
-->
\

#### Description of how you validated changes
I ran yarn test and got 100% coverage

#### Checklist

- [x] PR description included
- [x] `yarn test` passes

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/amazon-cognito-identity-js/__tests__/StorageHelperRN.test.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
+describe('Name of the group', () => {
```
Comment (nickarocho) [inline packages/amazon-cognito-identity-js/__tests__/StorageHelperRN.test.js:1]: Does this need an actual name?

File: packages/amazon-cognito-identity-js/jest.config.js [hunk 2]
```diff
@@ -27,7 +27,6 @@ module.exports = {
 				esModuleInterop: true,
 			},
 		},
-		__DEV__: true,
```
Comment (nickarocho) [inline packages/amazon-cognito-identity-js/jest.config.js:30]: What was this for?

**Review by nickarocho — CHANGES_REQUESTED** [review]
The describe block just needs a name, otherwise LGTM!

File: packages/amazon-cognito-identity-js/jest.config.js (hunk 2 above)
Comment (evcodes) [reply packages/amazon-cognito-identity-js/jest.config.js:30]: this was something for react native but I removed it since we don't need it anymore

**Pull request merged**